### DISCUSSION TITLE
What is this control in Phone Link app?

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [microsoft/WinUI-Gallery#1558] [Q&A] [answered]
![image](https://github.com/microsoft/WinUI-Gallery/assets/57910623/81d836ae-65fe-4ce3-94b3-0f40a858535a)

Stepper control in WinUI would be awesome!

**Comment by marcelwgn** [comment, marked as answer]
I don't think this a control that currently exists in WinUI 3. For feature proposals, the [WinUI repository](https://github.com/microsoft/microsoft-ui-xaml) is probably better suited

**Reply by DmitryBorodiy** [reply]
Awesome, I will create new feature request.

**Reply by DmitryBorodiy** [reply]
New feature request was created here : [https://github.com/microsoft/WinUI-Gallery/issues/1565#issue-2354738786](https://github.com/microsoft/WinUI-Gallery/issues/1565#issue-2354738786)
